src/Psalm/Internal/Analyzer/Statements/Expression/Assignment/PropertyAssignmentAnalyzer.php

```diff
@@ -237,6 +238,20 @@ public static function analyzeInstance(
                     continue;
                 }
 
+                if ($lhs_type_part instanceof Type\Atomic\TTemplateParam) {
+                    $extra_types = $lhs_type_part->extra_types;
```

Comment:
Do we need to assign this to a variable here? It seems it's only needed if we enter into the if on lines 250-251 and it just gets re-assigned.  Maybe able to do without it here.

Reply:
`$lhs_type_part` is overwritten on the line below. So `$extra_types` is reassigned, but to the property of a different object.